src/components/structures/RoomView.js

```diff
@@ -1690,6 +1691,8 @@ module.exports = React.createClass({
                         { previewBar }
                     </div>
                 );
+            } else {
+                hideRightPanel = true;
```

Comment:
The `else` block is a bit redundant, since the other branch returns... I can't decide if it's easier to read with or without though, since we're kind of in a maze of conditionals here... 😅

Reply:
:smile: quite the maze indeed. I usually find the redundant else a bit easier to read. I'll leave it in for now since you can't decide :)